packages/connection/src/class-manager.php

```diff
@@ -719,6 +719,21 @@ public function api_url( $relative_url ) {
 		$api_base = $api_base ? $api_base : 'https://jetpack.wordpress.com/jetpack.';
 		$version  = '/' . Utils::get_jetpack_api_version() . '/';
 
+		/**
+		 * Filters whether the connection manager should use the iframe authorization
+		 * flow instead of the regular redirect-based flow.
+		 *
+		 * @since 8.3.0
+		 *
+		 * @param Boolean $is_iframe_flow_used should the iframe flow be used, defaults to true.
+		 */
+		$iframe_flow = apply_filters( 'jetpack_use_iframe_authorization_flow', true );
```

Comment:
Do you have any particular reason to make the iframe the default within the package? I am fighting with myself on what would be a more appropriate default for now. 

On one hand, it feels a tad risky, since our in-place connection flow is not running for any sites currently, as the flow is not optimized yet. We don't want any accidental iframe connections happening in the wild right now, and making it `false` by default feels a bit safer. 

On the other hand, this in-place flow makes more sense for raw consumers of the package without Jetpack, as it bypasses all the custom URL stuff that is needed in the traditional flow. 

any thoughts @kbrown9 ?